pkg/hope/entities.go

```diff
@@ -157,7 +173,12 @@ func (node *Node) IsNode() bool {
 	return node.Role == "node" || node.IsMasterAndNode()
 }
 
+// IsHypervisor - Whether or not this node is a hypervisor node.
+func (node *Node) IsHypervisor() bool {
+	return node.Role == "hypervisor"
+}
+
 // IsRoleValid - Whether or not the node has a role that has been implemented.
 func (node *Node) IsRoleValid() bool {
```

Comment:
Should probably introduce something like `IsKubernetesRole()` or something, to keep the different behaviour sets separated properly.